NeoSharp.TransactionManager/NEP-5/INEP5.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace NeoSharp.TransactionManager.NEP_5
+{
+    interface INEP5
+    {
+		String TotalSupply();
+		String Name();
+		String Symbol();
+		String Decimals();
```

Comment:
String => string
TotalSupply, Decimal and Balance return BigInteger
Transfer return bool 
Please ensure it on NEP proposals